lib/ansible/modules/cloud/vmware/vca_vapp.py

```diff
@@ -229,6 +234,55 @@ def set_state(module):
         if not vapp.undeploy(action):
             module.fail('unable to undeploy vapp')
 
+def connect_to_network(module, vdc_name, vapp_name, network_name, network_mode):
+
+    nets = filter(lambda n: n.name == network_name, module.vca.get_networks(vdc_name))
+    assert len(nets) == 1
+    the_vdc = module.vca.get_vdc(vdc_name)
+    the_vapp = module.vca.get_vapp(the_vdc, vapp_name)
+    assert the_vapp
+    assert the_vapp.name == vapp_name
+
+    # Connect vApp
+    task = the_vapp.connect_to_network(nets[0].name, nets[0].href)
+    result = module.vca.block_until_completed(task)
+    assert result
+
+    # Connect VM
+    if(network_mode == 'pool'):
```

Comment:
yes, static would match to manual in pyvcloud where the IP is not allocate by cloud but provided by the user.
There has been a feature request on this and I'll probably add this capability on an upcoming patch once I'm finished developing a module to manage (add/delete) disks attached to a VM.